examples/reference/streams/plotly/Bounds.ipynb

```diff
@@ -67,6 +68,37 @@
    "source": [
     "<center><img src=\"https://assets.holoviews.org/gifs/examples/streams/plotly/bounds_selection.gif\" width=400></center>"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This example can also be displayed using [Plotly Dash](https://dash.plotly.com/):\n",
+    "\n",
+    "```python\n",
+    "from holoviews.plotting.plotly.dash import holoviews_to_dash\n",
+    "import dash\n",
+    "import dash_html_components as html\n",
+    "\n",
+    "# Create App\n",
+    "external_stylesheets = ['https://codepen.io/chriddyp/pen/bWLwgP.css']\n",
+    "app = dash.Dash(__name__, external_stylesheets=external_stylesheets)\n",
+    "\n",
+    "# Dash display\n",
+    "components = holoviews_to_dash(\n",
+    "    app, [layout], reset_button=True\n",
+    ")\n",
+    "\n",
+    "app.layout = html.Div(\n",
+    "    [components.graphs[0], components.resets[0], components.store]\n",
```

Comment:
I don't suppose there's a way to eliminate this step?  Is it not appropriate for `to_dash` to modify `app` to do that already, at least as a default (with the default overridden in some examples that need special widgets)? 

Overall, this is a lot of code to be adding to each of the examples, when most of it seems boilerplate.  Can we not instead just make a single page in the user guide that shows one Panel example, one Voila example, and one Dash example (plus the caveat that Dash will only work for the Plotly backend), and then have all the reference notebooks (for all backends) link back to that page? That seems easier for us to maintain plus throws less code at the user to try to follow (assuming any given user will click on many of these pages).

Reply:
> I don't suppose there's a way to eliminate this step? Is it not appropriate for to_dash to modify app to do that already, at least as a default (with the default overridden in some examples that need special widgets)?

I don't think so, the idea is that user should place them in the layout of a larger Dash app that includes other Dash components.

I'll make some comments about the docs down below